server/controllers/user.js

```diff
@@ -1,4 +1,9 @@
+const { JsonWebTokenError } = require('jsonwebtoken');
+const UserDao = require('../dao/user');
 const ApiError = require('../error/api-error');
+const jwt = require('jsonwebtoken');
```

Comment:
Not sure if this will be our coding standards, but I've often seen acronym variable names as being frowned upon (unless the acronym is obvious). It can be confusing if you have to repeatedly look up what they stand for.

Reply:
JWT is a pretty common acronym iirc. I've seen it with that acronym at my last workplace :)